etl/dbconn/connection.go

```diff
@@ -14,11 +14,14 @@ import (
 	"github.com/insolar/block-explorer/configuration"
 )
 
+// Connect returns connection to database
 func Connect(cfg configuration.DB) (*gorm.DB, error) {
```

Comment:
May be here you can provide your stop channel, to separate cmd and whole app.

Reply:
not here. I implement a bit different